pkg/apis/core/v1/zz_generated.validations.go

```diff
@@ -38,20 +37,11 @@ func init() { localSchemeBuilder.Register(RegisterValidations) }
 // RegisterValidations adds validation functions to the given scheme.
 // Public to allow building arbitrary schemes.
 func RegisterValidations(scheme *runtime.Scheme) error {
-	scheme.AddValidationFunc((*corev1.ReplicationController)(nil), func(ctx context.Context, op operation.Operation, obj, oldObj interface{}, subresources ...string) field.ErrorList {
-		if len(subresources) == 0 {
-			return Validate_ReplicationController(ctx, op, nil /* fldPath */, obj.(*corev1.ReplicationController), safe.Cast[*corev1.ReplicationController](oldObj))
-		}
```

Comment:
> We will support validation of subresources that share a kind by making the validation conditional.

I'm not seeing where this happens in this PR.  When we remove this conditional check, I'm not clear on what stops the validation from running on all ReplicationController endpoints regardless of subresource.

Reply:
Good catch. I'll put these checks back in place until we have ratcheting landed.